DeployToQA.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+APP_NAME=$1
+DOCKER_USER=$2
+DOCKER_PASS=$3
+REGISTRY=$4
+CLEAN_BRANCH_NAME=$5
+
+export CLEAN_BRANCH_NAME=$CLEAN_BRANCH_NAME # set this env variable for docker compose
+
+echo "Deploy to ${APP_NAME}-qa.satelapps.com" 
```

Comment:
Should be changed

Reply:
changed to?
Do you mean to [issue ](https://github.com/SatelCreative/DevOps-IT/issues/240#event-9070909096)you just created?

Reply:
The QA domain is `satelapps.com`???